README.md

```diff
@@ -1,38 +1,145 @@
-# Filter Employee Modal
+# Zadanie Rekrutacyjne - Michał Skrzypczyński
 
-## Description
+Plusy:
+- Wersja live
+- Readme
+- Fajne użycie styled-components z propsami
+- Proptypes
+- Custom hooks, np. z wielkością okna
 
-A simple website that filters employees by filters selected by a user.
-Made for recruitment purposes.
 
-## Preview
+Minusy:
+- config z pracownikami w App.js ?
```

Comment:
Przyświecała mi myśl, że App.js jako, że jest tak jakby najłatwiej dostępnym plikiem, jest dobrym miejscem dla takiego configu. W razie gdybyście chcieli dodać jakieś nowe dane pracowników to od razu idziecie do App.js i aktualizujecie dane. Jak tak teraz pomyślę to jest to całkiem wątpliwa oszczędność czasu, niewarta zaśmiecania tego pliku ;)

Reply:
Tak, App.js jest kluczowym plikiem każdej aplikacji. Wrzucanie takich rzeczy tam jest niefortunne. Lepsze miejsce zdecydowanie to constans albo jakis plik z mockiem Api Reply np .